src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj

```diff
@@ -131,6 +131,11 @@
       <Name>DynamoCore</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\Libraries\PythonNodeModelsWpf\PythonNodeModelsWpf.csproj">
+      <Project>{01DE9B06-0BCB-4D8A-862E-E8170F5D6B4F}</Project>
+      <Name>PythonNodeModelsWpf</Name>
+      <Private>False</Private>
+    </ProjectReference>
```

Comment:
From what I could see, this dependency is not really needed. The line that casts the parent window to a specific type `var parentWindow = Window.GetWindow(btn) as ScriptEditorWindow;` is not really necessary so we can remove this.